adcc/adc_pp/state2state_transition_dm.py

```diff
@@ -20,12 +20,119 @@
 ## along with adcc. If not, see <http://www.gnu.org/licenses/>.
 ##
 ## ---------------------------------------------------------------------
+from adcc import block as b
 from adcc.AdcMethod import AdcMethod
+from adcc.functions import einsum
 from adcc.AmplitudeVector import AmplitudeVector
+from adcc.OneParticleOperator import OneParticleOperator
 
 import libadcc
 
-DISPATCH = {}  # None implemented
+
```

Comment:
Thanks for extending that (and no probs it does not yet have CVS). In general I think you are too much copying libadc, i.e. I feel these equations could be compressed better by using dummy temporary variables or more explicit symmetrise etc. Remember since our evaluation is lazy temporary assignments don't cost because they don't evaluate.